include/SFML/Window/Event.hpp

```diff
@@ -389,6 +389,19 @@ class Event
 
     template <typename T>
     static constexpr bool isEventSubtype = isInParameterPack<T>(decltype (&m_data)(nullptr));
+
+    template<class T, class ...Ts>
```

Comment:
```suggestion
    template<class Callable, class ...Ts>
```
Let's name these callable template parameters. The template argument name `T` is already used in this file to refer to event subtypes so let's try to disambiguate a bit.